mkdocs.yml

```diff
@@ -1,41 +1,40 @@
 site_name: ElectionGuard
 # site_description:
-site_author: Microsoft
+# site_author: Microsoft
 # google_analytics:
 # remote_branch: for gh-deploy to GithubPages
 # remote_name: for gh-deploy to Github Pages
-copyright: "© Microsoft 2021"
+# copyright: "© Microsoft 2021"
 docs_dir: "docs"
 repo_url: https://github.com/microsoft/electionguard/
 nav:
   - About: 
     - Overview: index.md
     - Roadmap: Roadmap.md
-  - Guide:
-    - Overview: guide/Overview.md
-    - SDK: 
-      - Core Component: guide/Core_Component.md
-      - Implementations: guide/Implementations.md
-    - Additional Topics:
-      - Verifiable Election: guide/Verifiable_Election.md
-      - Election Manifest: guide/Election_Manifest.md
-      - Precinct Scan Use Case: guide/Precinct_scan.md
-      - Glossary: guide/Glossary.md 
-  - Specification:
-    - Overview: spec/Overview.md
-    - v0.95:
-      - Overview: spec/0.95.0/1_Overview.md
-      - Structure: spec/0.95.0/2_Structure.md
-      - Baseline Parameters: spec/0.95.0/3_Baseline_parameters.md     
-      - Key Generation: spec/0.95.0/4_Key_generation.md     
-      - Ballot Encryption: spec/0.95.0/5_Ballot_encryption.md          
-      - Ballot Aggregation: spec/0.95.0/6_Ballot_aggregation.md     
-      - Verifiable Decryption: spec/0.95.0/7_Verifiable_decryption.md     
-      - The Election Record: spec/0.95.0/8_The_election_record.md     
-      - Verifier Construction: spec/0.95.0/9_Verifier_construction.md 
-      - Applicability to E2E Verifiability and RLA: spec/0.95.0/10_Applicability.md
-      - Acknowledgements: spec/0.95.0/11_Acknowledgements.md
+    - Contributions: Contributions.md
+    - Glossary: Glossary.md  
+  - Specification v0.95:
+    - Overview: spec/0.95.0/1_Overview.md
+    - Structure: spec/0.95.0/2_Structure.md
+    - Baseline Parameters: spec/0.95.0/3_Baseline_parameters.md     
+    - Key Generation: spec/0.95.0/4_Key_generation.md     
+    - Ballot Encryption: spec/0.95.0/5_Ballot_encryption.md          
+    - Ballot Aggregation: spec/0.95.0/6_Ballot_aggregation.md     
+    - Verifiable Decryption: spec/0.95.0/7_Verifiable_decryption.md     
+    - The Election Record: spec/0.95.0/8_The_election_record.md     
+    - Verifier Construction: spec/0.95.0/9_Verifier_construction.md 
+    - Applicability to E2E Verifiability and RLA: spec/0.95.0/10_Applicability.md
+    - Acknowledgements: spec/0.95.0/11_Acknowledgements.md
     - v0.85: "https://github.com/microsoft/electionguard/releases/tag/v0.85.0"
+  - Developers:
```

Comment:
This navigation is super confusing from my perspective.

1. The `glossary` in the about but then `how is electionguard works` includes a blurb about what electionguard is?
2. The top level tabs range from specificity. 

![Screen Shot 2021-03-01 at 2 01 22 PM](https://user-images.githubusercontent.com/10125297/109545254-9ea18280-7a96-11eb-9f66-6d587e3e57ca.png)

I think there is overall a navigation issue with this setup in comparison with other libraries that do the same thing. This needs to be compared against other developer literature:
https://stripe.com/docs/api
https://www.twilio.com/docs/usage/api
https://www.dropbox.com/developers/documentation
https://reactjs.org/docs/getting-started.html

Reply:
I would suggest this

1. `About` 
2. `Guide` 
3. `Specification`
4. `Community`

Then within the guide:
1. Overview (with links to other sections)
2. Develop
      A. Getting Started
3. Use Cases
4. Glossary